data/json/items/comestibles/carnivore.json

```diff
@@ -309,7 +309,7 @@
     "healthy": -2,
     "quench": 2,
     "fun": -10,
-    "calories": 208,
+    "calories": 50,
```

Comment:
I thought this looked really low, but it turns out that lung tissue has really low mass because it's less dense than most flesh.  Realistically, both mass and calories should increase by about 50%.

Reply:
I have no problem with increasing the weight and volume and adjusting for the change in calories.

Reply:
relevant PR is #26922

Reply:
There's no need to increase the volume.  250ml of lung tissue should weigh around 80 g.  Not a big deal to mess with KorGgenT's PRs.